XpringKit/Org_Xrpl_Rpc_V1_IssuedCurrencyAmount+XRPIssuedCurrency.swift

```diff
@@ -0,0 +1,13 @@
+import BigInt
+
+internal extension XRPIssuedCurrency {
+  init?(issuedCurrency: Org_Xrpl_Rpc_V1_IssuedCurrencyAmount) {
+    guard let value = BigInt(issuedCurrency.value) else {
+      return nil
+    }
+
+    self.currency = XRPCurrency(currency: issuedCurrency.currency)
+    self.value = value
+    self.issuer = issuedCurrency.issuer.address
```

Comment:
what would happen if issuedCurrency.issuer didn't exist or didn't have an address?  Is this impossible?

Reply:
It's impossible based on API constraints. This proto will just default to the empty string if it didn't exist.